.runtests.sh

```diff
@@ -8,6 +8,9 @@ go get -t -v ./... || exit 1
 echo "Testing engo.io/engo using coveralls"
 $HOME/gopath/bin/goveralls -service=travis-ci
 
+echo "Testing engo.io/engo using 'go test'"
```

Comment:
Change to "Testing engo.io/engo using 'go test -v -bench'" or something of that nature (to mark a difference between regular tests and benchmarks.